SrijanShovit/HealthLearning
# Alzheimer's Commit

### Description
A clear and concise description of what the PR does.
- This PR does the following:
  - Adds a new feature for predicting Alzheimer's disease using deep learning techniques.
  - Fixes issues related to accessibility and user interface for better usability.
  - Updates the app to include processing of MRI scans or other relevant medical imaging data.

### Related Issues
Link any related issues using the format `Fixes #159 `.
- Placeholder: "Fixes #159 "

### Changes
List the detailed changes made in this PR.
- Added a new feature to predict Alzheimer's disease using the ADNI dataset.
- Refactored the existing code to integrate the new machine learning models for Alzheimer's prediction.

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Run the Jupyter notebook to ensure the model training and validation are functioning correctly.
3. Execute `app.py` to launch the application.
4. Verify that the app can process and provide predictions accurately.
5. Check that the new user interface elements are accessible and user-friendly.

### Additional Context
- This PR is based on the need for an accessible tool for early detection of Alzheimer's disease, utilizing the ADNI dataset for model training and validation.

### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
